frontend/test/metabase/scenarios/onboarding/about.cy.spec.js

```diff
@@ -0,0 +1,20 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("scenarios > about Metabase", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.visit("/");
+    cy.icon("gear").click();
+    cy.findByText("About Metabase").click();
+  });
+
+  it.skip("should display correct Metabase version (metabase#15656)", () => {
+    cy.findByText(/You're on version v[01]\.\d+\.\d+(-SNAPSHOT)?/i);
```

Comment:
There are historic cases of v1.2.3.4 - and we should probably allow something more generic in the end (RC1 etc).
```suggestion
    cy.findByText(/You're on version v[01](\.\d+){2,3}(-[\w\d]+)?/i);
```